src/utils/index.ts

```diff
@@ -24,3 +26,25 @@ export const renameKeys: RenameKeys<Dictionary<any>, Dictionary<string>> =
     Object.keys(o)
       .reduce((acc, k) => assoc(map[k] || k, o[k], acc), {})
 );
+
+export const pipeToPromise = (stream: stream.Readable): Promise<Buffer[]> =>
```

Comment:
curious of why this this is called `pipeToPromise`?

Reply:
Streams have a method `pipe` which allows you to write the output of a `Readable` stream directly into a `Writable` stream. But I want to `pipe` the contents of a `Readable` stream into memory, and simply be notified (via promise resolution) when that's done.

Reply:
Can this be renamed as `streamPipeToPromise` for clarity?

Reply:
`pipe` is the verb here, so it should be first really. Can do `pipeStreamToPromise` if you like.